src/Dfe.PlanTech.Domain/Content/Models/ContentSupport/ContentSupportPage.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System.Diagnostics.CodeAnalysis;
+using Dfe.PlanTech.Domain.Content.Interfaces;
 
-namespace Dfe.PlanTech.Web.Models.Content;
+namespace Dfe.PlanTech.Domain.Content.Models.ContentSupport;
 
 [ExcludeFromCodeCoverage]
-public class ContentSupportPage : ContentBase, IContentSupportPage<Entry>
+public class ContentSupportPage : ContentBase, IContentSupportPage<CSBodyText>
 {
-    public Heading Heading { get; init; } = null!;
-    public List<Entry> Content { get; init; } = [];
+    public CSHeading Heading { get; init; } = null!;
+    public List<CSBodyText> Content { get; init; } = [];
```

Comment:
Dont necessarily need to change this here, but why List over Enumerable?

Reply:
This is how it was originally and also seems consistent with how Contentful models are handled across PT, so I left it